tests/two_d_primitive_tests.cpp

```diff
@@ -0,0 +1,106 @@
+#include "two_d_primitive_tests.h"
+
+#include <SDL_image.h>
```

Comment:
I think this code no longer uses SDL.

Reply:
good catch, intellij had the includes collapsed on. me.  removed.

Reply:
I'm thinking maybe I don't need vertexbuffer.h or debugoutput.h either?  @abaire am I good to remove those as well?

Reply:
Yeah, please remove anything that's not actually used. I think CLion does a reasonably good job at greying out unused headers, so you can just delete them.

Reply:
done